### PR TITLE
[es] - Updated `kubectl` Installation Docs

### LINKED ISSUE #27078: [es] update `kubectl` installation docs
**What would you like to be added**

The current documentation upstream documentation for `kubectl` installation is divided into a document per system (mac, linux, windows). The Spanish documentation still uses the old format, will a single page for all systems and is a bit outdated.

- en: https://kubernetes.io/docs/tasks/tools/#kubectl
- es: https://k8s.io/es/docs/tasks/tools/install-kubectl/

**Why is this needed**

`kubectl` installation is probably the first step when onboarding into the kubernetes ecosystem and the documentation should be up to date.

**Comments**

Discussed on https://github.com/kubernetes/website/issues/27048 .
Added to the #13948 .

---
/language es
/good-first-issue
/help

### PR DESCRIPTION
**Pull request by adithyaakrishna**
Signed-off-by: Adithya Krishna <aadithya794@gmail.com>

**Summary:**
This PR Updates the `kubectl `installation docs in Spanish

**Description:**
This PR Fixes #27078

### CODE REVIEW

File: content/es/docs/tasks/tools/included/install-kubectl-linux.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+weight: 10
+card:
+  name: tasks
+  weight: 20
+  title: Instalar kubectl en Linux
+---
+
+## {{% heading "prerequisites" %}}
+
+Debes usar una versión de kubectl que esté dentro de una diferencia de versión menor de tu clúster. Por ejemplo, un v{{< skew latestVersion >}} La cliente puede comunicarse con v{{< skew prevMinorVersion >}}, v{{< skew latestVersion >}}, y v{{< skew nextMinorVersion >}} aviones de control.
```
[the remark this answers is not in the document]
Comment (adithyaakrishna) [reply content/es/docs/tasks/tools/included/install-kubectl-linux.md:14]: Okay sure @electrocucaracha will fix this up ASAP 👍🏻

File: content/es/docs/tasks/tools/included/kubectl-convert-overview.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Descripción general de kubectl-convert"
+description: >-
+  Un complemento de kubectl que le permite convertir manifiestos de una versión
+  de una API de Kubernetes a una versión diferente.
+headless: true
+---
+
+Un complemento para la herramienta de línea de comandos de Kubernetes `kubectl`, que le permite convertir manifiestos entre diferentes versiones de API. Esto puede ser particularmente útil para migrar manifiestos a una versión de API no obsoleta con la versión más reciente de Kubernetes.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/kubectl-convert-overview.md:9]:
```suggestion
Un plugin para la herramienta de línea de comandos de Kubernetes `kubectl`, que le permite convertir manifiestos entre diferentes versiones de la API. Esto puede ser particularmente útil para migrar manifiestos a una versión no obsoleta de la API con la versión más reciente de Kubernetes.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Autocompletar bash en Linux"
+description: "Alguna configuración opcional para la finalización automática de bash en Linux."
+headless: true
+---
+
+### Introducción
+
+El script de finalización de kubectl para Bash se puede generar con el comando `kubectl completion bash`. Obtener el script de finalización en su shell habilita el autocompletado de kubectl.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:9]:
```suggestion
El script de completado de kubectl para Bash se puede generar con el comando `kubectl completion bash`. Obtener el script de completado en su shell habilita el autocompletado de kubectl.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+Sin embargo, el guión de finalización depende de [**bash-completion**](https://github.com/scop/bash-completion), lo que significa que primero debe instalar este software (puedes probar si tienes bash-completion ya instalado ejecutando `type _init_completion`).
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:11]:
```suggestion
Sin embargo, el script de completado depende de [**bash-completion**](https://github.com/scop/bash-completion), lo que significa que primero debe instalar este software (puedes probar si tienes bash-completion ya instalado ejecutando `type _init_completion`).
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+### Instalar bash-complete
+
+La finalización de bash es proporcionado por muchos administradores de paquetes (ver [aquí](https://github.com/scop/bash-completion#installation)). Puedes instalarlo con `apt-get install bash-completion` o `yum install bash-completion`, etc.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:15]:
```suggestion
El completado de bash es proporcionado por muchos administradores de paquetes (ver [aquí](https://github.com/scop/bash-completion#installation)). Puedes instalarlo con `apt-get install bash-completion` o `yum install bash-completion`, etc.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+Los comandos anteriores crean `/usr/share/bash-completion/bash_completion`, que es el guión principal de bash-complete. Dependiendo de su administrador de paquetes, debe obtener manualmente este archivo en su `~/.bashrc` expediente.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:17]:
```suggestion
Los comandos anteriores crean `/usr/share/bash-completion/bash_completion`, que es el script principal de bash-complete. Dependiendo de su administrador de paquetes, debe obtener manualmente este archivo de perfil en su `~/.bashrc`.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+```bash
+source /usr/share/bash-completion/bash_completion
+```
+
+Vuelva a cargar su shell y verifique que la finalización de bash esté correctamente instalada escribiendo `type _init_completion`.
+
+### Habilitar el autocompletado de kubectl
+
+Ahora debe asegurarse de que el script de finalización de kubectl se obtenga en todas sus sesiones de shell. Hay dos formas de hacer esto:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:29]:
```suggestion
Ahora debe asegurarse de que el script de completado de kubectl se obtenga en todas sus sesiones de shell. Hay dos formas de hacer esto:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+- Obtenga el script de finalización en su `~/.bashrc` expediente:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:31]:
```suggestion
- Obtenga el script de completado en su perfil `~/.bashrc`:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+   ```bash
+   echo 'source <(kubectl completion bash)' >>~/.bashrc
+   ```
+
+- Agregue el guión de finalización al `/etc/bash_completion.d` directorio:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:37]:
```suggestion
- Agregue el script de completado al directorio de `/etc/bash_completion.d`:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+
+   ```bash
+   kubectl completion bash >/etc/bash_completion.d/kubectl
+   ```
+
+Si tiene un alias para kubectl, puede extender la finalización del shell para trabajar con ese alias:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:43]:
```suggestion
Si tiene un alias para kubectl, puede extender el completado del shell para trabajar con ese alias:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+
+```bash
+echo 'alias k=kubectl' >>~/.bashrc
+echo 'complete -F __start_kubectl k' >>~/.bashrc
+```
+
+{{< note >}}
+fuentes de bash-complete todos los guiones `/etc/bash_completion.d`.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:51]:
```suggestion
El código fuente de bash-complete todos los scripts se encuentra en `/etc/bash_completion.d`.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 12]
```diff
@@ -0,0 +1,88 @@
+---
+title: "Autocompletar bash en macOS"
+description: "Alguna configuración opcional para la finalización automática de bash en macOS."
+headless: true
+---
+
+### Introducción
+
+El script de finalización de kubectl para Bash se puede generar con `kubectl completion bash`. Obtener este script en su shell permite la finalización de kubectl.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:9]:
```suggestion
El script de completado de kubectl para Bash se puede generar con `kubectl completion bash`. Obtener este script en su shell permite el completado de kubectl.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+
+Sin embargo, el script de finalización de kubectl depende de [**finalización de bash**](https://github.com/scop/bash-completion) que, por lo tanto, debe instalar previamente.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:11]:
```suggestion
Sin embargo, el script de finalización de kubectl depende de [**bash-completion**](https://github.com/scop/bash-completion) que, por lo tanto, debe instalar previamente.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+
+{{< warning>}}
+Hay dos versiones de bash-complete, v1 y v2. V1 es para Bash 3.2 (
+que es el predeterminado en macOS), y v2 es para Bash 4.1+. El script de finalización de kubectl **no funciona** correctamente con bash-complete v1 y Bash 3.2. Requiere **bash-complete v2** y **Bash 4.1+**. Por lo tanto, para poder usar correctamente la finalización de kubectl en macOS, debe instalar y usar Bash 4.1+ ([*instrucciones*](https://itnext.io/upgrading-bash-on-macos-7138bd1066ba)). Las siguientes instrucciones asumen que usa Bash 4.1+ (es decir, cualquier versión de Bash de 4.1 o posterior).
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:15]:
```suggestion
que es el predeterminado en macOS), y v2 es para Bash 4.1+. El script de completado de kubectl **no funciona** correctamente con bash-complete v1 y Bash 3.2. Requiere **bash-complete v2** y **Bash 4.1+**. Por lo tanto, para poder usar correctamente la finalización de kubectl en macOS, debe instalar y usar Bash 4.1+ ([*instrucciones*](https://itnext.io/upgrading-bash-on-macos-7138bd1066ba)). Las siguientes instrucciones asumen que usa Bash 4.1+ (es decir, cualquier versión de Bash de 4.1 o posterior).
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 15]
```diff
@@ -0,0 +1,88 @@
...
+{{< /warning >}}
+
+### Actualizar Bash
+
+Las instrucciones aquí asumen que usa Bash 4.1+. Puede verificar la versión de su Bash ejecutando:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:20]:
```suggestion
Las siguientes instrucciones asumen que usa Bash 4.1+. Puede verificar la versión de su Bash ejecutando:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 16]
```diff
@@ -0,0 +1,88 @@
...
+
+```bash
+echo $BASH_VERSION
+```
+Si es demasiado antiguo, puede instalarlo / actualizarlo usando Homebrew:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:25]:
```suggestion
Si es demasiado antiguo, puede instalarlo o actualizarlo usando Homebrew:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 17]
```diff
@@ -0,0 +1,88 @@
...
+```bash
+echo $BASH_VERSION $SHELL
+```
+
+Homebrew generalmente lo instala en `/usr/local/bin/bash`.
+
+### Instalar bash-complete
+
+{{< note >}}
+Como se mencionó, estas instrucciones asumen que usa Bash 4.1+, lo que significa que instalará bash-completacion v2 (a diferencia de Bash 3.2 y bash-deployment v1, en cuyo caso la finalización de kubectl no funcionará).
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:41]:
```suggestion
Como se mencionó antes, estas instrucciones asumen que usa Bash 4.1+, lo que significa que instalará bash-completacion v2 (a diferencia de Bash 3.2 y bash-deployment v1, en cuyo caso el completado de kubectl no funcionará).
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 18]
```diff
@@ -0,0 +1,88 @@
...
+{{< /note >}}
+
+Puede probar si ya tiene instalado bash-complete v2 con `type _init_completion`.Si no es así, puede instalarlo con Homebrew:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:44]:
```suggestion
Puede probar si ya tiene instalado bash-complete v2 con `type _init_completion`. Si no es así, puede instalarlo con Homebrew:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 19]
```diff
@@ -0,0 +1,88 @@
...
+
+```bash
+brew install bash-completion@2
+```
+
+Como se indica en el resultado de este comando, agregue lo siguiente a su archivo `~ / .bash_profile`:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:50]:
```suggestion
Como se indica en el resultado de este comando, agregue lo siguiente a su archivo `~/.bash_profile`:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 20]
```diff
@@ -0,0 +1,88 @@
...
+
+```bash
+export BASH_COMPLETION_COMPAT_DIR="/usr/local/etc/bash_completion.d"
+[[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"
+```
+
+Vuelva a cargar su shell y verifique que bash-complete v2 esté instalado correctamente con`type _init_completion`.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:57]:
```suggestion
Vuelva a cargar su shell y verifique que bash-complete v2 esté instalado correctamente con `type _init_completion`.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 21]
```diff
@@ -0,0 +1,88 @@
...
+
+### Habilitar el autocompletado de kubectl
+
+Ahora debe asegurarse de que el script de finalización de kubectl se obtenga en todas sus sesiones de shell. Hay varias formas de lograrlo:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:61]:
```suggestion
Ahora debe asegurarse de que el script de completado de kubectl se obtenga en todas sus sesiones de shell. Hay varias formas de lograrlo:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 22]
```diff
@@ -0,0 +1,88 @@
...
+
+- Obtenga el script de finalización en su `~/.bash_profile` expediente:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:63]:
```suggestion
- Obtenga el script de finalización en su perfil `~/.bash_profile`:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 23]
```diff
@@ -0,0 +1,88 @@
...
+
+    ```bash
+    echo 'source <(kubectl completion bash)' >>~/.bash_profile
+    ```
+
+- Agregue el guión de finalización al `/usr/local/etc/bash_completion.d` directorio:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:69]:
```suggestion
- Agregue el script de completado al directorio `/usr/local/etc/bash_completion.d`:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 24]
```diff
@@ -0,0 +1,88 @@
...
+
+    ```bash
+    kubectl completion bash >/usr/local/etc/bash_completion.d/kubectl
+    ```
+
+- Si tiene un alias para kubectl, puede extender la finalización del shell para trabajar con ese alias:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:75]:
```suggestion
- Si tiene un alias para kubectl, puede extender el completado del shell para trabajar con ese alias:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 25]
```diff
@@ -0,0 +1,88 @@
...
+
+    ```bash
+    echo 'alias k=kubectl' >>~/.bash_profile
+    echo 'complete -F __start_kubectl k' >>~/.bash_profile
+    ```
+
+- Si instaló kubectl con Homebrew (como se explica [aquí](/docs/tasks/tools/install-kubectl-macos/#install-with-homebrew-on-macos)), entonces el script de finalización de kubectl ya debería estar en `/usr/local/etc/bash_completion.d/kubectl`. En ese caso, no necesita hacer nada.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:82]:
```suggestion
- Si instaló kubectl con Homebrew (como se explica [aquí](/docs/tasks/tools/install-kubectl-macos/#install-with-homebrew-on-macos)), entonces el script de completado de kubectl ya debería estar en `/usr/local/etc/bash_completion.d/kubectl`. En ese caso, no necesita hacer nada.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 26]
```diff
@@ -0,0 +1,88 @@
...
+
+   {{< note >}}
+   La instalación de Homebrew de bash -mentation v2 genera todos los archivos en el `BASH_COMPLETION_COMPAT_DIR` directorio, es por eso que los dos últimos métodos funcionan.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:85]:
```suggestion
   La instalación de Homebrew de bash-completion v2 genera todos los archivos en el directorio `BASH_COMPLETION_COMPAT_DIR`, es por eso que los dos últimos métodos funcionan.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 27]
```diff
@@ -0,0 +1,88 @@
...
+   {{< /note >}}
+
+En cualquier caso, después de recargar su shell, la finalización de kubectl debería estar funcionando.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:88]:
```suggestion
En cualquier caso, después de recargar su shell, el completado de kubectl debería estar funcionando.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-zsh.md [hunk 28]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Autocompletar zsh"
+description: "Alguna configuración opcional para la finalización automática de zsh."
+headless: true
+---
+
+El script de finalización de kubectl para Zsh se puede generar con el comando `kubectl completion zsh`. Obtener el script de finalización en su shell habilita el autocompletado de kubectl.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-zsh.md:7]:
```suggestion
El script de completado de kubectl para Zsh se puede generar con el comando `kubectl completion zsh`. Obtener el script de completado en su shell habilita el autocompletado de kubectl.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-zsh.md [hunk 29]
```diff
@@ -0,0 +1,29 @@
...
+
+Para hacerlo en todas sus sesiones de shell, agregue lo siguiente a su`~/.zshrc` expediente:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-zsh.md:9]:
```suggestion
Para hacerlo en todas sus sesiones de shell, agregue lo siguiente a su perfil `~/.zshrc`:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-zsh.md [hunk 30]
```diff
@@ -0,0 +1,29 @@
...
+
+```zsh
+source <(kubectl completion zsh)
+```
+Si tiene un alias para kubectl, puede extender la finalización del shell para trabajar con ese alias:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-zsh.md:14]:
```suggestion
Si tiene un alias para kubectl, puede extender el completado del shell para trabajar con ese alias:
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-zsh.md [hunk 31]
```diff
@@ -0,0 +1,29 @@
...
+
+```zsh
+echo 'alias k=kubectl' >>~/.zshrc
+echo 'complete -F __start_kubectl k' >>~/.zshrc
+```
+
+Después de recargar su shell, el autocompletado de kubectl debería estar funcionando.
+
+Si recibe un error como `complete:13: command not found: compdef`, 
+luego agregue lo siguiente al comienzo de su `~/.zshrc` expediente:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-zsh.md:24]:
```suggestion
luego agregue lo siguiente al comienzo de su perfil `~/.zshrc`:
```

File: content/es/docs/tasks/tools/included/verify-kubectl.md [hunk 32]
```diff
@@ -0,0 +1,31 @@
+---
+title: "verificar la instalación de kubectl"
+description: "Cómo verificar kubectl."
+headless: true
+---
+
+Para que kubectl encuentre y acceda a un clúster de Kubernetes, necesita un
+[archivo kubeconfig](/docs/concepts/configuration/organize-cluster-access-kubeconfig/),que se crea automáticamente cuando creas un clúster usando
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/verify-kubectl.md:8]:
```suggestion
[archivo kubeconfig](/docs/concepts/configuration/organize-cluster-access-kubeconfig/), que se crea automáticamente cuando creas un clúster usando
```

File: content/es/docs/tasks/tools/included/verify-kubectl.md [hunk 33]
```diff
@@ -0,0 +1,31 @@
...
+Verifique que kubectl esté configurado correctamente obteniendo el estado del clúster:
+
+```shell
+kubectl cluster-info
+```
+Si ve una respuesta de URL, kubectl está configurado correctamente para acceder a su clúster.
+
+Si ve un mensaje similar al siguiente, kubectl no está configurado correctamente o no puede conectarse a un clúster de Kubernetes.
+```
+La conexión al servidor <server-name:port> fue rechazada. ¿Especificó el host o puerto correcto?
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/verify-kubectl.md:22]:
```suggestion
The connection to the server <server-name:port> was refused - did you specify the right host or port?
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

**Comment by jimangel** [comment]
@adithyaakrishna are you still working on this? If so, would you mind resolving the conflicting files and [squashing your commits](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits)?

If PR is open for greater than a month without action, the next PR wrangler may close it if appropriate (and you can return / reopen when ready to continue).

Thanks!

**Comment by raelga** [comment]
@adithyaakrishna  can you check the latests comments?

**Comment by adithyaakrishna** [comment]
@raelga @jimangel Thanks for reminding :)
Yes sure, Apologies for the delay. I have addressed most of the changes but some are pending still. Will finish it up ASAP

File: content/es/docs/tasks/tools/included/install-kubectl-linux.md [hunk 34]
```diff
@@ -0,0 +1,244 @@
...
+
+## Verificar la configuración de kubectl
+
+{{< include "verify-kubectl.md" >}}
+
+## Complementos y configuraciones opcionales de kubectl
+
+### Habilitar el autocompletado de shell
+
+kubectl proporciona soporte de autocompletado para Bash y Zsh, lo que puede ahorrarle mucho tiempo al escribir.
```
Comment (raelga) [inline content/es/docs/tasks/tools/included/install-kubectl-linux.md:178]:
```suggestion
kubectl proporciona soporte de autocompletado para Bash y Zsh, lo que puede ahorrarle mucho tiempo al interactuar con la herramienta.
```

**Review by raelga — COMMENTED** [review]
@adithyaakrishna let me know if you need any help with the review process!

**Comment by adithyaakrishna** [comment]
@electrocucaracha @raelga Apologies for the delay :/
I have made all the requested changes to the PR and Rebased the branch as well. Please do lmk if you have any changes/suggestions as well :)

File: content/es/docs/tasks/tools/included/install-kubectl-linux.md (hunk 1 above)
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/install-kubectl-linux.md:14]:
```suggestion
Debes usar una versión de kubectl que esté dentro de una diferencia de versión menor de tu clúster. Por ejemplo, un cliente v{{< skew latestVersion >}} puede comunicarse con v{{< skew prevMinorVersion >}}, v{{< skew latestVersion >}}, y v{{< skew nextMinorVersion >}} del plano de control.
```

File: content/es/docs/tasks/tools/included/install-kubectl-linux.md [hunk 35]
```diff
@@ -0,0 +1,244 @@
...
+   ```
+
+   {{< note >}}
+   Si no tiene acceso de root en el sistema de destino, aún puede instalar kubectl en el `~/.local/bin` directorio:
+
+   ```bash
+   chmod +x kubectl
+   mkdir -p ~/.local/bin/kubectl
+   mv ./kubectl ~/.local/bin/kubectl
+   # y luego agrega ~/.local/bin/kubectl para $PATH
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/install-kubectl-linux.md:87]:
```suggestion
   # y luego agregue ~/.local/bin/kubectl en el $PATH
```

File: content/es/docs/tasks/tools/included/install-kubectl-macos.md [hunk 36]
```diff
@@ -0,0 +1,248 @@
...
+weight: 10
+card:
+  name: tasks
+  weight: 20
+  title: Instalar kubectl en macOS
+---
+
+## {{% heading "prerequisites" %}}
+
+Debes usar una versión de kubectl que esté dentro de una diferencia de versión menor de tu clúster. Por ejemplo, un v{{< skew latestVersion >}} El cliente puede comunicarse con v{{< skew prevMinorVersion >}}, v{{< skew latestVersion >}}, y v{{< skew nextMinorVersion >}} del plano de control.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/install-kubectl-macos.md:14]:
```suggestion
Debes usar una versión de kubectl que esté dentro de una diferencia de versión menor de tu clúster. Por ejemplo, un cliente v{{< skew latestVersion >}} puede comunicarse con v{{< skew prevMinorVersion >}}, v{{< skew latestVersion >}}, y v{{< skew nextMinorVersion >}} del plano de control.
```

**Review by electrocucaracha — COMMENTED** [review]
/lgtm

/cc @raelga

**Review by anyulled — APPROVED** [review]

**Pull request merged**